ext/thymeleaf/src/main/java/org/eclipse/krazo/ext/thymeleaf/CDIWebContext.java

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2014-2015 Oracle and/or its affiliates. All rights reserved.
- * Copyright (c) 2018, 2019 Eclipse Krazo committers and contributors
+ * Copyright (c) 2018, 2019, 2022 Eclipse Krazo committers and contributors
```

Comment:
Just a minor one. AFAIK the Eclipse format for the copyright is `<first_year>, <last_year>`. So there is no need to enumerate all the years. So basically like this:

```suggestion
 * Copyright (c) 2018, 2022 Eclipse Krazo committers and contributors
```

@ivargrimstad Is that correct or do I misunderstand the default convention?